CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - `CloudProperties` now throw `ResourceLoadingException` when loading properties from a
   source that does not exist; the thrown excepion wraps its cause
 - fixed preliminary closing of SSH client when using `asDaemon` method of the `ExecBuilder`
+- added support for Docker volume bindings; It's configured through a new `DockerNode`
+  property `docker.binds` which accepts comma separated list of bindings
```

Comment:
OK, we already have `docker.portBindings`, so I'll change this to `docker.volumeBindings`.